package.json

```diff
@@ -6,17 +6,17 @@
   "license": "MIT",
   "scripts": {
     "dev": "nodemon --config config/nodemon/dev.json",
-    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development babel-node node_modules/.bin/webpack-dev-server --config config/webpack/webpack.client.babel.ts",
+    "dev:nowatch": "cross-env NODE_ENV=development BABEL_ENV=development webpack-dev-server --config config/webpack/webpack.client.babel.ts",
     "dev:clean": "rimraf '../.build/development/{*,.*}' 'node_modules/.cache'",
-    "prod": "cross-env NODE_ENV=production BABEL_ENV=production babel-node node_modules/.bin/webpack --config config/webpack/webpack.client.babel.ts",
+    "prod": "cross-env NODE_ENV=production BABEL_ENV=production webpack --config config/webpack/webpack.client.babel.ts",
     "prod:clean": "rimraf '../.build/production/{*,.*}' 'node_modules/.cache'",
     "prod:serve": "nodemon --quiet --exec babel-node --extensions '.ts' ./src/server/server.ts",
     "prod:watch": "nodemon --config config/nodemon/prod.json",
-    "lint": "yarn eslint; yarn stylelint; yarn tsc; yarn tslint",
-    "lint:fix": "yarn eslint:fix; yarn tsc; yarn tslint:fix",
+    "lint": "yarn eslint && yarn stylelint && yarn tsc && yarn tslint",
+    "lint:fix": "yarn eslint:fix && yarn tsc && yarn tslint:fix",
```

Comment:
This is not the same behavior. This will stop  after the first command that errors out.
I'd prefer to see all the errors, in both dev and CI environment.

I think we could just add spaces around semicolons. That should unscrew Windows.

Reply:
Hey @ivan-aksamentov ,

the line: 
`node -e 'console.clear()' ; yarn dev:clean ; yarn install && yarn dev:nowatch || cd .`
doesn't properly work for me on Windows (neither with Powershell nor default CMD shell, maybe with a bash shell?)

it actually runs just the part after && (yarn dev:nowatch, etc) and skips the part before (clear, dev:clean, install).

yes, it properly runs the server etc but ';' doesn't seem to work properly on Win (at least for me).

Reply:
If you want, you can play with
https://www.npmjs.com/package/npm-run-all

in particular, `run-s` with `-c` flag when appropriate
https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-s.md

Let's hope it won't cause too many troubles on Unix.

Reply:
@ivan-aksamentov 
It simplifies "lint" scripts quite a bit:
```
"lint": "run-s -c eslint stylelint tsc tslint",    
"lint:fix": "run-s -c eslint:fix tsc tslint:fix",
```
but clutters a bit more "dev" ("prod") scripts as

`"exec": "run-s -c clear dev:clean dev:start || cd ."` in `dev.json`

needs definitely adding "dev:start" and even "clear" to `package.json`:
`"dev:start": "yarn install && yarn dev:nowatch",`

Maybe it is a bit cleaner this way, on the other hand this `exec` line  could be run even  sequentially with exit on error:
`"exec": "node -e 'console.clear()' && yarn dev:clean && yarn install && yarn dev:nowatch || cd ."`
to compromise complexity.

Reply:
@mickeypearce Mickey, absolutely, open a PR! And thanks for the investigation.